2-mandatory/1-create-functions.js

```diff
@@ -3,15 +3,19 @@ Write a function that:
 - Accepts an array as a parameter.
 - Returns a new array containing the first five elements of the passed array.
 */
-function first5() {
+function first5(inputArray) {
+  firstFiveNumbersArr = inputArray.slice(0, 5);
+  return firstFiveNumbersArr;
 }
 
 /*
 Write a function that:
 - Accepts an array as a parameter.
 - Returns a new array containing the same elements, except sorted.
 */
-function sortArray() {
+function sortArray(inputArray) {
+  let sortedArr = inputArray.sort();
+  return sortedArr;
 }
```

Comment:
Well done !! 
Please be aware that sort() method sorts the elements of an array and returns the reference to the same array. 
Which means it will sort inputArray. To avoid this, you can try to copy the inputArray as value then sort the new array. 
sortedArr = [...inputArray]; //it will copy the value of inputArray and not a reference to it. 
sortedArr.sort( ); 

 [https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/sort](url)
 
 Hope that helps. 😊
 Keep up the good work 👌👌